specification/v2/bindings/restful/spec.md

```diff
@@ -107,10 +107,10 @@ paths:
           description: Unique identifier of the ecosystem governing authority being recognized.
           schema:
             type: string
-        - name: authority_id
+        - name: ecosystem_id
```

Comment:
since this is ecosystem_id, are we specifiying this somewhere.

Reply:
I messed up on that spec.md, which is not actually part of the SpecUp rendering. I think it should be deleted.